src/v3/test/integration/__snapshots__/authenticator-verification-okta-verify-push-code.test.tsx.snap

```diff
@@ -482,7 +511,36 @@ exports[`authenticator-verification-okta-verify-push-code should render form 1`]
                     class="MuiBox-root emotion-29"
                     data-se="icon-code"
                   >
-                    <mocksvgicon />
+                    <svg
+                      aria-hidden="true"
+                      aria-labelledby="phoneIcon"
+                      fill="none"
+                      height="63"
+                      role="img"
+                      viewBox="0 0 51 63"
+                      width="51"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <title
+                        id="phoneIcon"
+                      >
+                        L10N_ERROR[icon.title.mobileDevice]
```

Comment:
Just marking here to follow back up after running the `yarn build:release` to ensure it has been fixed.

Reply:
yup that fixed it 👍